src/Database/Adapter/SQL.php

```
@@ -1834,68 +1835,57 @@ public function getTenantQuery(
      * Get the SQL projection given the selected attributes
      *
-     * @param array<string> $selections
-     * @param string $prefix
-     * @param array<string> $spatialAttributes
-     * @return mixed
+     * @param array<Query> $selects
+     * @return string
      * @throws Exception
      */
-    protected function getAttributeProjection(array $selections, string $prefix, array $spatialAttributes = []): mixed
+    protected function getAttributeProjection(array $selects, array $spatialAttributes = []): string
```

PHPMD (2.15.0)

Avoid unused parameters such as '$spatialAttributes'. (Unused Code Rules)

(UnusedFormalParameter)